geoips/plugins/modules/output_checkers/image.py

```diff
@@ -165,40 +161,38 @@ def outputs_match(plugin, output_product, compare_product, threshold=0.05):
     # and exact diff image to log, for easy viewing.  Return False.
     if thresholded_retval != 0:
         bad_inds = np.where(diff_arr != 0)
-        LOG.interactive("    ***************************************")
-        LOG.interactive("    *** BAD Images do NOT match within tolerance ***")
-        LOG.interactive("    *** ***")
-        LOG.interactive(
-            "    *** %s mismatched pixels exceeding threshold %s ***",
-            thresholded_retval,
-            threshold,
+        log_with_emphasis(LOG.interactive, ["BAD Images do NOT match within tolerance"])
```

Comment:
This line should also be included in the below call to log_with_emphasis, not a separate call, please.